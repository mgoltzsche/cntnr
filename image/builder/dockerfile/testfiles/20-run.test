FROM alpine:3.7
RUN ["/bin/echo", "hello from container"]
RUN ["/bin/echo", "hello from container"]
RUN echo "hello world" > /etc/hello
RUN echo "special characters & whitespaces" > '/äh so'

# ASSERT RUN [ -f '/etc/hello' ] && [ -f '/äh so' ]
# ASSERT ERR false