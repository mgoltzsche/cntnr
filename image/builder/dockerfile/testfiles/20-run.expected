FROM alpine:3.7
RUN "/bin/echo" "hello from container"
RUN "/bin/echo" "hello from container"
RUN "/bin/sh" "-c" "echo \"hello world\" > /etc/hello"
RUN "/bin/sh" "-c" "echo \"special characters & whitespaces\" > '/Ã¤h so'"